pkg/controller/migmigration/task.go

```diff
@@ -1011,6 +1011,23 @@ func (t *Task) init() error {
 			Durable:  true,
 		})
 	}
+
+	someArtifactAvailable, msg, err := t.hasRelevantArtifacts()
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	if !someArtifactAvailable && !t.rollback() {
```

Comment:
Also add `!t.Stage()` to the comparison here. Stage migrations are already considered above with the `StageNoOp` warning, so we don't want to compare with the mig analytics numbers for stage migrations.